doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -542,6 +546,20 @@ Enabling and disabling notations
 
       Scopes are not compatible with :ref:`abbreviations <Abbreviations>`.
 
+Here are examples:
+
+.. coqtop:: all
+
+   Disable Notation "+" (all).
+
+.. coqtop:: all
```

Comment:
Are these lines really needed (e.g. to get a blank line in the output)?  Same thing just below.

Reply:
I don't think this is blank in the output. According to the artifact, `all` seems to have the effect of showing the output of Coq.

Reply:
I guess Jim's question was rather: why not put all three commands under the same `.. coqtop:: all` block?

Reply:
Ah, I see. This is because I thought that otherwise the outputs would come after the block of commands but if it is interspersed, then these lines are indeed not needed. I removed them then.